app/views/site/facultyFilterSelection.html.haml

```diff
@@ -4,15 +4,15 @@
     %title Taulbee Survey Data
     :css
       .heading{
-      	color:white;
-      	background-color:#500000;
-      	height:50px;
-      	margin:0;
-      	padding:0;
+        color:white;
```

Comment:
These are just formatting changes, please try to avoid submitting these changes in future.